src/server/scripts/Outland/zone_netherstorm.cpp

```diff
@@ -949,6 +949,32 @@ class spell_q10190_battery_recharging_blaster_aura : public AuraScript
     }
 };
 
+enum Veraku
+{
+    NPC_VERAKU               = 18544,
+    SPELL_CHALLENGE_VERAKU   = 34895
+};
+
+class spell_challenge_veraku : public SpellScript
+{
+public:
+    PrepareSpellScript(spell_challenge_veraku);
+
```

Comment:
Need to validate spells that are used in scripts

Reply:
SPELL_CHALLENGE_VERAKU

Reply:
I don't get it

Reply:
```cpp
    bool Validate(SpellInfo const* /*SpellEntry*/) override
    {
        return ValidateSpellInfo({ SPELL_CHALLENGE_VERAKU });
    }
```

Reply:
Is this normal for the condition?

Reply:
Everytime we use a spell within a script we need to validate that it exists.

Reply:
So this time as you use `!veraku->HasAura(SPELL_CHALLENGE_VERAKU)` you need to first validate that this spell exists before we can execute the script

Reply:
See https://github.com/azerothcore/azerothcore-wotlk/blob/123bf37ee7e6763d9d7f4b778cfe8453577bce6c/src/server/scripts/Spells/spell_quest.cpp#L2108-L2123 for example

Reply:
understood. Thank you